core/includes/common.inc

```diff
@@ -5565,13 +5725,13 @@ function backdrop_add_library($module, $name, $every_page = NULL) {
  * - Two (or more) modules can still register the same library and use it
  *   without conflicts in case the libraries are loaded on certain pages only.
  *
- * @param $module
+ * @param string $module
  *   The name of a module that registered a library.
- * @param $name
+ * @param string|null $name
  *   (optional) The name of a registered library to retrieve. By default, all
  *   libraries registered by $module are returned.
  *
- * @return
+ * @return array
```

Comment:
This can return FALSE if a specific one's requested and the library doesn't exist / isn't found.